client-resttemplate/swagger.gradle

```diff
@@ -4,6 +4,13 @@ dependencies {
 
 	implementation 'io.swagger.core.v3:swagger-annotations:2.1.13'
 	swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.33'
+
+	// Transitive dependency constraints due to security vulnerabilities in prior versions.
+	// These are not directly included, they are just constrained if they are pulled in as
+	// transitive dependencies.
+	constraints {
+		implementation('org.yaml:snakeyaml:1.31')
```

Comment:
Last time I did deps updates sourceclear flagged snakeYAML versions under 2.0 as unsafe and there was a whole rigamarole with snakeYAML 2.0 requiring spring boot 3 which broke a bunch of other things. Does 1.31 fix the vulnerability without requiring a spring boot upgrade?